frontend/src/SearchBar.css

```diff
@@ -0,0 +1,4 @@
+.container{
```

Comment:
I think it might be better for this file to be in the src/Components folder

Reply:
You can create a new folder under src to store css files